File: BSc_CompSc/3rdyr/unix/c++/sum.cpp
#include<iostream>

using namespace std;

int main()
{
	int n,i,s=0;
	cout<<"Enter n:- ";
	cin>>n;
	for(i=1;i<=n;i++)
	{
		s=s+i;
	}
	cout<<"SUM = "<<s<<endl
		<<"AVG = "<<s/n;
	return 0;
}

File: MSc_CompSc/2nd_sem/graphics/workspacce/mou/src/java/jhilik/java programs 1 (jhilik)/Sum.java
import java.util.Scanner;
public class Sum
{
	public static void main(String args[])
	{
	Scanner input = new Scanner(System.in);	
	int range,number,sum=0,i;
	System.out.println("Enter the range");
	range=input.nextInt();
	for(i=1;i<=range;i++)
	{
		System.out.println("Enter the number");	
		number=input.nextInt();
		sum=sum+number;
	}
	System.out.println("The sum is:"+sum);
	}
}